src/components/animations/appearings.tsx

```diff
@@ -26,3 +26,26 @@ export const fadeUp = keyframes({
 export const FadeUp = styled('div')({
   animation: `${fadeUp} 300ms forwards`,
 })
+
+export const slideUp = (maxHeight: number, initialMaxHeight = 0) =>
+  keyframes({
+    from: {
+      maxHeight: initialMaxHeight,
+    },
+    to: {
+      maxHeight,
```

Comment:
Just FYI the easing gets messed up if you don't use the exact height of the element as maxHeight, as it will interpolate in a curve going from say 0 to 300, so it might not look as good as it could if you would have the exact height. 

I would say that you should render the element off screen first to get the exact height :)

Reply:
Good point, but after an irl discussion i think we will push this for later :)